tests/unit/layout/size_of.rs

```diff
@@ -5,5 +5,5 @@
 use layout::Fragment;
 use layout::SpecificFragmentInfo;
 
-size_of_test!(test_size_of_fragment, Fragment, 160);
+size_of_test!(test_size_of_fragment, Fragment, 136);
 size_of_test!(test_size_of_specific_fragment_info, SpecificFragmentInfo, 24);
```

Comment:
Why? The size of `Fragment` does not actually change and unit tests fail because of this.